superset/views/base_api.py

```diff
@@ -127,7 +127,7 @@ class BaseSupersetModelRestApi(ModelRestApi):
         "data": "list",
         "delete": "delete",
         "distinct": "list",
-        "export": "mulexport",
+        "export": "list",
```

Comment:
@dpgaspar, can you help me with this? I had to change this to `list` in order to make the new export work, but I assume this will break the old export (which I want to keep for now). What is `mulexport` here?

Reply:
there is you can bring this mapping to the leaf class itself. But why do we want a new permission name(it's already mapped to it's class domain) ?

Reply:
I think I'm misunderstanding how this works, and I think I also need `mulexport` for databases (so we can select multiple dbs and export them at once). Let me take another look.